### ISSUE TITLE
Resetting `typoTolerance` does not reset `minWordSizeForTypos`

### ISSUE THREAD

**Issue by maryamsulemani97** [meilisearch/meilisearch#2330]
**Describe the bug**
Resetting the typo tolerance settings does not set `minWordSizeForTypos` to the default values

**To Reproduce**

1. Update the `typoTolerance` object:
```
curl \
    -X POST 'http://localhost:7700/indexes/movies/settings/typo-tolerance' \
    -H 'Content-Type: application/json' \
    --data-binary '{
          "disableOnAttributes": ["title"],
          "disableOnWords": ["shrek"],
          "minWordLengthForTypo": {
              "oneTypo": 8,
              "twoTypos": 10
    }
}'
```
2. Verify if the update was made:

![Screen Shot 2022-04-18 at 20 54 12](https://user-images.githubusercontent.com/90181761/163843303-f0d2a954-9ed5-41ad-a368-af48890fb6a2.png)

3. reset typo tolerance:
```
curl \
    -X GET 'http://localhost:7700/indexes/movies/settings/typo-tolerance'
```

**Expected behavior**
Should have reset the entire object

**What happened**
The `minWordSizeForTypos` object did not reset 

**Screenshots**

![Screen Shot 2022-04-18 at 21 03 16](https://user-images.githubusercontent.com/90181761/163844386-278f5407-fbb4-43f3-afc6-a45c68266f6e.png)

**Meilisearch version:** v0.27.0rc1

**Comment by MarinPostma** [comment]
Good catch! the bug is on milli's side, fixing it now, Thanks!

**Comment by curquiza** [comment]
Can we add tests to ensure the reset behavior is correctly applied?

**Comment by curquiza** [comment]
Fixed by milli#499 and #2335
